src/secrets.js

```diff
@@ -0,0 +1,3 @@
+// Replace the below values with respective write key and dataplane URL on local
```

Comment:
Should we leave a comment mentioning that this file appears in the browser and should be used cautiously?

Reply:
Sounds good, will add more context.

Reply:
Added more context and warning message